frontend_vue/src/components/ui/IncidentDetails.vue

```diff
@@ -81,7 +81,7 @@
                                 !isObject(deepestnested_val) &&
                                 isNotEmpty(deepestnested_val)
                               "
-                              :label="deepnested_key"
+                              :label="deepernested_key"
```

Comment:
Does this not affect other tokens?

Reply:
It likely does, so I ran through it with SRK to double check my thinking. 
The issue here is that the data structure we're trying to render is 
```
{
    'coords': {
        'accuracy': 1,
        'latitude': 0,
        'longitude': 0,
        ...
    },
    'timestamp': 0
}
```
and this code is essentially trying to parse that nested structure and output the labels. Without this change we get `coords` as the label for latitude, longitude, accuracy, etc.:

<img width="500" alt="Screenshot 2024-07-31 at 11 08 56" src="https://github.com/user-attachments/assets/90b82594-a12c-4f44-86d2-3a73c8cc36d0">


On reflection we should check redirect tokens in particular to see if they're being passed an idiosyncratic datastructure... I remember something about lists of values passed from the browser, so it's not just dictionaries all the way down.

Reply:
This has been fixed :)